rust/rsnano/src/voting/inactive_cache_information.rs

```diff
@@ -0,0 +1,45 @@
+use crate::core::{Account, BlockHash};
+use crate::voting::InactiveCacheStatus;
+use std::time::{Duration, SystemTime, UNIX_EPOCH};
+
+/// Information on the status of inactive cache information
+#[derive(Clone)]
+pub struct InactiveCacheInformation {
+    pub arrival: SystemTime,
```

Comment:
Rusts equivalent for C++'s `steady_clock` is `Instant`. `SystemTime` could jump back in time